NetSSL_OpenSSL/samples/SetSourceIP/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 set(SAMPLE_NAME "SetSourceIP")
 
+# uncomment following line, might solve compiling issues
+#set(CMAKE_CXX_STANDARD 11)
```

Comment:
this should not be needed, since there is automatic [detection](https://github.com/pocoproject/poco/blob/develop/cmake/CXX1x.cmake)
@Bjoe is cmake maintainer, you may want to check with him